### PR TITLE
Correct error-details lookup in response from "message" to "Message"

### PR DESCRIPTION
**Pull request by jdhood1**

### CODE REVIEW

**Review by motobob — APPROVED** [review]

**Review by nalundgaard — APPROVED** [review]
Nice. Oh, title case!

**Comment by motobob** [comment]
@jdhood1 would you mind fixing up broken tests too and it's good to go.

File: test/erlcloud_ddb_tests.erl [hunk 1]
```diff
@@ -19,7 +19,7 @@
 -define(_f(F), fun() -> F end).
 
 -export([validate_body/2]).
-
```
Comment (jdhood1) [inline test/erlcloud_ddb_tests.erl:22]: It looks like my editor cleaned up some trailing white spaces.  Sorry about that.

**Comment by jdhood1** [comment]
> @jdhood1 would you mind fixing up broken tests too and it's good to go.

Sorry about that.  The tests passed in my application, so my brain checked out.  I just attempted to fixed them.  Hopefully, those changes are correct.

File: test/erlcloud_ddb_tests.erl (hunk 1 above)
Comment (motobob) [reply test/erlcloud_ddb_tests.erl:22]: thats fine for tests. not fine for actual code as it pollutes the git history

**Pull request merged**